doc/rados/configuration/osd-config-ref.rst

```diff
@@ -553,24 +560,103 @@ words, the share of each type of service is controlled by three tags:
 #. weight: the proportional share of capacity if extra capacity or system
    oversubscribed.
 
-In Ceph, operations are graded with "cost". And the resources allocated for
-serving various services are consumed by these "costs". So, for example, the
-more reservation a services has, the more resource it is guaranteed to possess,
-as long as it requires. Assuming there are 2 services: recovery and client ops:
+In Ceph operations are graded with "cost". And the resources allocated
+for serving various services are consumed by these "costs". So, for
+example, the more reservation a services has, the more resource it is
+guaranteed to possess, as long as it requires. Assuming there are 2
+services: recovery and client ops:
 
 - recovery: (r:1, l:5, w:1)
 - client ops: (r:2, l:0, w:9)
 
-The settings above ensure that the recovery won't take never more than 5 units
-of resources even if it requires so, and no other services are competing with
-it. But if the clients start to issue large amount of I/O requests, neither
-will they exhaust all the I/O resources. 1 unit of resources is always
-allocated for recovery jobs. So the recovery jobs won't be starved even in
-a cluster with high load. And in the meantime, the client ops can enjoy
-a larger portion of the I/O resource, because its weight is "9", while its
-competitor "1". In the case of client ops, it is not clamped by the limit
-setting, so it can make use of all the resources if there is no recovery
-ongoing.
+The settings above ensure that the recovery won't get more than 5
```

Comment:
i think the reservation will be updated with the request's cost if the request is to be served. and the cost is different from one request to another. so can we equate the unit of reservation/limit to the number of requests?